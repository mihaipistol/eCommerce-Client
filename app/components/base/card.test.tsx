import { render, screen } from '@testing-library/react';
import Card from './card';

describe('Card', () => {
  it('renders children correctly', () => {
    render(<Card>Test Content</Card>);
    expect(screen.getByText('Test Content')).toBeDefined();
  });

  it('applies custom class names', () => {
    render(<Card className='custom-class'>Test Content</Card>);
    const cardElement = screen.getByText('Test Content').closest('div');
    expect(cardElement).toMatchSnapshot();
  });
});
