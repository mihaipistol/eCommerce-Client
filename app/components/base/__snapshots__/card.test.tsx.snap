// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card applies custom class names 1`] = `
<div
  class="min-w-sm md:min-w-md lg:min-w-lg p-8 bg-white dark:bg-black shadow-xl rounded-lg ring-1 ring-slate-900/5 custom-class"
>
  Test Content
</div>
`;
